components/multi-platform-controller/production/stone-prd-rh01/host-values.yaml

```diff
@@ -229,9 +229,58 @@ dynamicConfigs:
 
   linux-test-amd64:
     ami: ami-01aaf1c29c7e0f0af
+    instance-type: "m6idn.2xlarge"
+    sudo-commands: "/usr/bin/podman, /usr/bin/rm /usr/share/containers/mounts.conf"
+    disk: "200"
+    user-data: |-
+      Content-Type: multipart/mixed; boundary="//"
+      MIME-Version: 1.0
+
+      --//
+      Content-Type: text/cloud-config; charset="us-ascii"
+      MIME-Version: 1.0
+      Content-Transfer-Encoding: 7bit
+      Content-Disposition: attachment; filename="cloud-config.txt"
+
+      #cloud-config
+      cloud_final_modules:
+        - [scripts-user, always]
+
+      --//
+      Content-Type: text/x-shellscript; charset="us-ascii"
+      MIME-Version: 1.0
+      Content-Transfer-Encoding: 7bit
+      Content-Disposition: attachment; filename="userdata.txt"
+
+      #!/bin/bash -ex
+
+      # Format and mount NVMe disk
+      mkfs -t xfs /dev/nvme1n1
+      mount /dev/nvme1n1 /home
+
+      # Create required directories
+      mkdir -p /home/var-lib-containers /var/lib/containers /home/var-tmp /var/tmp /home/ec2-user/.ssh
+
+      # Setup bind mounts
+      mount --bind /home/var-lib-containers /var/lib/containers
+      mount --bind /home/var-tmp /var/tmp
+
+      # Configure ec2-user SSH access
+      chown -R ec2-user /home/ec2-user
+      sed -n 's,.*\(ssh-.*\s\),\1,p' /root/.ssh/authorized_keys > /home/ec2-user/.ssh/authorized_keys
+      chown ec2-user /home/ec2-user/.ssh/authorized_keys
+      chmod 600 /home/ec2-user/.ssh/authorized_keys
+      chmod 700 /home/ec2-user/.ssh
+      restorecon -r /home/ec2-user
+
+      --//--
 
   linux-test-arm64:
     ami: ami-06f37afe6d4f43c47
+    sudo-commands: "/usr/bin/podman, /usr/bin/rm /usr/share/containers/mounts.conf"
```

Comment:
same here, sudo-commands, iops and throughput are missing in template, c/p the from any root templ

Reply:
Done